src/restrictededitingexceptionui.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module restricted-editing/restrictededitingexceptionui
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+
+import restrictedDocumentIcon from '../theme/icons/restricted.svg';
```

Comment:
Not sure this is the best name of the icon file (`restricted.svg`). It should rather describe what it depicts rather than what feature uses it (except obvious ones like bold maybe), for instance "pilcrow.svg" not "paragraph.svg". It makes them reusable. But if you don't find anything better, just ignore it :)

Reply:
OMG it will be hard... the icon was used as a stub since Damian already provided it to me I wasn't too attached to it anyway. :)

I don't have a better name than `restricedEditingMode` as it was decided that will use it in that mode. The other idea is: `lockedDocument` do describe the icon contents (I've discarded the `lockOnContent`)

WDYT?

Reply:
`contentlock.svg` maybe?